WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -618,8 +626,15 @@ private extension DashboardViewController {
     /// Shows or hides the free trial banner.
     ///
     func observeFreeTrialBannerVisibility() {
-        viewModel.$freeTrialBannerViewModel.sink { [weak self] viewModel in
+        Publishers.CombineLatest(viewModel.$freeTrialBannerViewModel,
+                                 viewModel.$showOnboarding)
+        .sink { [weak self] viewModel, showOnboarding in
```

Comment:
❓ similar to above, from my understanding we just want to hide other header views above the onboarding card. the bottom banner shouldn't be affected